### ISSUE TITLE
Different language support

### ISSUE THREAD

**Issue by MaximeGodbout** [aaronparker/packagefactory#55]
Hi, I don't know if I'm asking this in the right section as I'm new to the Github environment:

Would it be possible to have a variable to change the language of the installers, if applicable, for applications when using PackageFactory? Or something that we could precise the installer's language and pick a different description (fitting the language picked)?

I'm going to use Mozilla Firefox as an example, here's what’s in the "**App.json**" for Mozilla Firefox:
`"Filter": "Get-EvergreenAppFromApi -Name \"MozillaFirefox\" -ErrorAction \"SilentlyContinue\" -WarningAction \"SilentlyContinue\" | Where-Object { $_.Channel -eq \"Current\" -and $_.Architecture -eq \"x64\" -and $_.Type -eq \"msi\" -and $_.Language -eq \"en-GB\" }",`

The installer is locked with the "**en-GB**" language. I would like the possibility to change it.

When I'm using Evergreen manually, I can use: 
`Get-EvergreenApp -Name "MozillaFirefox" -AppParams @{Language="fr"}`
And it will get the French installer.

I could change it manually through the "**App.json**" but then if I want to update the versioning of all the apps when using "**git pull**" on my local repo, it'll get replaced by "**en-GB**" again.

Maybe there's a better way to do it, I'm not familiar enough with "git" to know.

TLDR: I want to download/import applications in Intune with the help of PackageFactory but still being able to control which language is picked for the application, if applicable (also being able to modify the "description" field so my users can read the description with the language I picked) without it being reverted by updating the repo.

If I didn't post this in the right section, could you please direct me where I should?
Thank you very much for reading through this

**Comment by aaronparker** [comment]
You could do a couple of things:
* Fork the repo, update your copy of the Firefox `app.json`, and don't pull changes from this repo
* Create a new Firefox en-GB app package and create a pull request back to this repo, so that there are two Firefox packages

**Comment by MaximeGodbout** [comment]
Hey Aaron!
Thank you very much for replying! I have some questions about your ideas:
If I fork the repo but don't pull changes from the repo, I will not benefit from the updated versioning of the App.json that happens from your Github Workflow (Update Package Manifests)? 
If I create a different Firefox package, only 1 of the 2 packages will be updated? Unless your Update Package Manifests go through all the App.json and update only the versions + detection method?
